oss-cloud-backend/services/utility.js

```diff
@@ -19,3 +25,29 @@ module.exports.generateResponse = (status, body, stringify = true) => ({
   headers: { 'Access-Control-Allow-Origin': '*' },
   body: stringify ? JSON.stringify(body) : body,
 });
+
+
+module.exports.validateJSON = (data, schemaName) => {

```

Comment:
It seems this is a replacement for `module.exports.checkBody` function.
The `checkBody` can be removed if so.

Reply:
P.S. lodash import can be removed too.